x-pack/test/detection_engine_api_integration/utils/create_rule_saved_object.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type SuperTest from 'supertest';
+
+import { Rule } from '@kbn/alerting-plugin/common';
+import {
+  BaseRuleParams,
+  InternalRuleCreate,
+} from '@kbn/security-solution-plugin/server/lib/detection_engine/rule_schema';
+
+/**
```

Comment:
I thought about that, but was also trying to avoid hitting any possible circular dependency issues. Since the work to move over these tests all into `security_solution_api_integration` is a work in progress and should be resolved in the next weeks, I thought it ok to play it safe here and just duplicate.